prefect_hightouch/credentials/generated.py

```diff
@@ -39,7 +39,7 @@ class HightouchCredentials(Block):
     """
 
     _block_type_name = "Hightouch Credentials"
-    # _logo_url = "<LOGO_URL_HERE>"  # noqa
+    # _logo_url = "<UPDATE _logo_url IN __init__.py>"  # noqa
```

Comment:
I'm probably just lacking context with the generator, but why is this necessary?

Reply:
To not override user input